CodeStarterCamp_Week4/Routine.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Routine.swift
+//  CodeStarterCamp_Week4
+//
+//  Created by 정선아 on 2022/05/08.
+//
+
+import Foundation
+
+struct Routine {
+    let routineName: String
+    var exercises: Array<Exercise>
```

Comment:
Routine 구조체의 exercises 프로퍼티도 var → `let`으로 변경해도 문제없네요! 😄 
`배열`이 들어가니까, 단수형인 exercise 가 아니라 복수형인 exercises 사용하신 점도 정말 좋습니다!

콜론(:) 뒤에 타입을 명시해주실 때는 Array<Exercise> 라고 해주셨는데
`[Exercise]` 라는 표현을 쓰지 않는 이유가 있으실까요~?
저는 좀 더 간편해서 많이 쓰거든요! ㅎㅎ

```suggestion
    let routineName: String
    let exercises: [Exercise]
```

Reply:
배열 선언에 익숙하지 않아서 계속 `Array<>`형태를 사용하는 거 같습니다🥲
객체 파트를 공부하면서 `변수`, `상수`에 대한 개념도 모호해지는 거 같아요😵‍💫
복습 차원으로 `배열`과 `변수`, `상수` 부분 다시 보도록 할게요!

Reply:
이 이미지 한 번 참고해보세요!
출처는 [이 블로그](https://velog.io/@ryan-son/Swift-%EA%B5%AC%EC%A1%B0%EC%B2%B4%EC%99%80-%ED%81%B4%EB%9E%98%EC%8A%A4-%EA%B7%B8%EB%A6%AC%EA%B3%A0-%ED%94%84%EB%A1%9C%ED%8D%BC%ED%8B%B0%EC%9D%98-%EC%83%81%EC%88%98%EC%99%80-%EB%B3%80%EC%88%98-%EC%84%A0%EC%96%B8%EC%97%90-%EB%94%B0%EB%A5%B8-%EA%B0%92-%EB%B3%80%EA%B2%BD-%EA%B0%80%EB%8A%A5-%EC%97%AC%EB%B6%80%EC%97%90-%EB%8C%80%ED%95%9C-%EA%B3%A0%EC%B0%B0) 입니다! 😄 

![image](https://user-images.githubusercontent.com/71127966/167376036-9ecbb73b-ae23-40d5-aafb-79b93feb0c70.png)

![image](https://user-images.githubusercontent.com/71127966/167376051-535cb356-5563-46e1-bc4e-83578fa8f44d.png)